packages/shared/src/hooks/useFeedSettings.ts

```diff
@@ -53,14 +54,29 @@ export default function useFeedSettings({
   );
 
   const { tagsCategories, feedSettings, advancedSettings } = feedQuery;
+  const checkEnabledSettings = useCallback(
```

Comment:
Also the naming contradict it searches one value.
checkSettingsEnabledState or something might be better? (WDYT)

Reply:
Makes sense. Let's update it.